zarr/tests/test_hierarchy.py

```diff
@@ -492,6 +492,8 @@ def test_getattr(self):
         # test
         eq(g1['foo'], g1.foo)
         eq(g2['bar'], g2.bar)
+        # test that hasattr returns False instead of an exception (issue #88)
+        eq(hasattr(g1, '__unexistingattribute__'), False)
```

Comment:
Very minor nit-pick: it would be slightly more direct to use assert_false(hasattr(g1, '__unexistingattribute__')).